todo_app/tests_e2e/test_end_to_end.py

```diff
@@ -1,18 +1,17 @@
-import os, requests, pytest, app
+import os, pytest, app
+
 from threading import Thread
+from Mongo_db import delete_mongo_db
 
-from Trello_items import boardsurl, get_trello_key, get_trello_token, build_auth_query
-from Trello_boards import create_trello_board, delete_trello_board
 from dotenv import load_dotenv, find_dotenv
 
 @pytest.fixture(scope='module')
 def test_app():
-    # Create the new board & update the board id environment variable
+    # Create the new DB & update the DB environment variable
     file_path = find_dotenv('.env')
     load_dotenv(file_path, override=True)
-    board_id = create_trello_board()
-    os.environ['TRELLO_TODO_BOARDID'] = board_id
-
+    test_db_name = "testing_database"
+    os.environ['MONGO_DB'] = test_db_name
```

Comment:
Similarly to the integration test, here you're setting the environment variable after the import chain has set the relevant values on your application Config. A simple workaround would be to modify that config directly before the app loads it, e.g. here, rather than setting the environment variable, we can set:
```python
...

    test_db_name = "testing_database"
    Config.MONGO_DB = test_db_name
    # construct the new application
    application = app.create_app()
...
```